package.json

```diff
@@ -1,27 +1,34 @@
 {
-  "scripts": {
-    "build": "gulp",
-    "dev": "gulp serve",
-    "start": "http-server ."
-  },
-  "devDependencies": {
+  "name": "landing_page_preproduction",
+  "version": "1.0.0",
```

Comment:
why did you remove `scripts`?

Reply:
No special reason except my inattention.